web/src/right_sidebar_ui.js

```diff
@@ -13,14 +12,22 @@ export function show_userlist_sidebar() {
 }
 
 export function register_click_handlers() {
-    $("#userlist-toggle-button").on("click", (e) => {
-        e.preventDefault();
-        e.stopPropagation();
+    document.addEventListener(
+        "click",
+        (e) => {
```

Comment:
Hrm, I don't love registering an event handler that runs on every click; we should at least try to have this exit early after doing the cheapest check -- maybe a `return` right after fetching `sidebarHidden` if that's false.

Also should not use camel case for that variable.